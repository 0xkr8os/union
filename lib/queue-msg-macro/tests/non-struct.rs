use queue_msg_macro::msg_struct;

#[msg_struct]
enum NonStruct {
    One,
    Two,
}

fn main() {}
