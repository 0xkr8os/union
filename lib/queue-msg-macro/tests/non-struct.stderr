error: Only support Structs
 --> tests/non-struct.rs:3:1
  |
3 | #[msg_struct]
  | ^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `msg_struct` (in Nightly builds, run with -Z macro-backtrace for more info)

error: Only support Structs
 --> tests/non-struct.rs:9:1
  |
9 | #[msg_struct]
  | ^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `msg_struct` (in Nightly builds, run with -Z macro-backtrace for more info)
